Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -593,7 +594,56 @@ open class Kommunicate: NSObject,Localizable, KMPreChatFormViewControllerDelegat
         }
     }
     
-    public func userSubmittedResponse(name: String, email: String, phoneNumber: String, password: String) {
+    /**
+    launches the PreChat Lead Collection with custom payload
+
+     - Parameters:
+     - appID: User's application ID.
+     - inputList: list of LeadCollectionField objects to create a form in pre Chat.
+     - viewController: ViewController from which the pre-chat form view will be launched.
+     */
+    open class func launchPreChatWithCustomPayload(appID: String,viewController: UIViewController, inputList: [LeadCollectionField],completion: @escaping([String:String]?,KommunicateError?) -> ()){
```

Comment:
Space is missing b/w some arg names. Do SwiftFormat setup and run the tests, it will auto-format the code.

Reply:
Done the setup & formatted

Reply:
Can you add [this file](https://github.com/Kommunicate-io/KommunicateChatUI-iOS-SDK/blob/master/.swiftformat) to this project? Make sure you are formatting using this file to keep the rules consistent across projects.

Reply:
After doing these changes, run the formatter again.

Reply:
@Sathyan-Elangovan this is pending.

Reply:
done